fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingThrottle.java

```diff
@@ -84,15 +84,13 @@ public class IndexingThrottle {
     }
 
     public <R> CompletableFuture<R> buildCommitRetryAsync(@Nonnull BiFunction<FDBRecordStore, AtomicLong, CompletableFuture<R>> buildFunction,
-                                                          boolean limitControl,
```

Comment:
Why was this removed?
Is `IndexingByRecords.buildEndpoints` the only thing that was passing `false` here? Would that mean that it could retry building the endpoints more than it needs to?

Reply:
It was removed to simplify the code a bit. 
`IndexingByRecords.buildEndpoints` will retry just the same, but it *will* increase/decrease limit upon success/failure. It seems harmless to me, especially since  `IndexingByRecords` is now obsolete (single index is now built by the simpler `IndexingMultiTargetByRecords`). 

BTW, when mutual indexing is mature, I hope to retire `IndexingByRecords` altogether. It's only use now is for explicit parallel/range indexing.